spec/swagger_helper.rb

```diff
@@ -30,6 +30,248 @@
+                },
+              },
+              contract_type: {
+                type: :string,
+                enum: Vacancy.contract_types.keys,
+              },
+              phases: {
+                type: :array,
+                minItems: 1,
+                items: {
+                  type: :string,
+                  enum: Vacancy.phases.keys,
+                },
+              },
+              key_stages: {
+                type: :array,
+                minItems: 1,
+                items: {
+                  type: :string,
+                  enum: Vacancy.key_stages.keys,
+                },
+              },
+              subjects: {
+                type: :array,
+                minItems: 1,
+                items: {
+                  type: :string,
+                  enum: ["Accounting",
+                         "Art and design",
+                         "Biology",
+                         "Business Studies",
+                         "Chemistry",
+                         "Citizenship",
+                         "Classics",
+                         "Computing",
+                         "Dance",
+                         "Design And Technology",
+                         "Drama",
+                         "Economics",
+                         "Engineering",
+                         "English",
+                         "Food Technology",
+                         "French",
+                         "Geography",
+                         "German",
+                         "health_and_social_care",
+                         "history",
+                         "Humanities",
+                         "ICT",
+                         "Languages",
+                         "Law",
+                         "Mandarin",
+                         "Mathematics",
+                         "Media Studies",
+                         "Music",
+                         "Philosophy",
+                         "Physical Education",
+                         "Physics",
+                         "PSHE",
+                         "Psychology",
+                         "Religious Education",
+                         "Science",
+                         "Social Science",
+                         "Sociology",
+                         "Spanish",
+                         "Statistics"],
+                },
+              },
+            },
+          },
+          unauthorized_error: {
+            type: "object",
+            properties: {
+              errors: {
+                type: "array",
+                items: {
+                  type: "object",
+                  properties: {
+                    error: { type: "string", example: "unauthorized" },
+                    message: { type: "string", example: "Invalid API key" },
+                  },
+                  required: %w[error message],
+                },
+              },
+            },
+          },
+          not_found_error: {
```

Comment:
I duplicated all these schemas with the same structure to have individualized examples per error.

I believe once the API gets implemented and the request specs properly set, a single error schema would cover these cases, and the autogenerated examples for each error response could be set in the test file.